supplementary_style_guide/glossary_terms_conventions/product_conventions/partner-and-hybrid-cloud-infrastructure.adoc

```diff
@@ -0,0 +1,762 @@
+[[partner-and-hybrid-cloud-infrastructure-conventions]]
+
+[discrete]
+[[access-mode]]
+==== image:images/yes.png[yes] access mode (noun)
+*Description*: The concurrency method used when accessing a storage component, such as `readwriteonce` or `readwritemany`.
+
+*Use it*: Yes
```

Comment:
The "yes" "no" and "with caution" should be lowercase. Can you sweep through the PR to fix these?

Reply:
Done, thankyou!